_posts/2021/2021-09-20-compact_blank-remove-empty-strings-from-array-and-hash.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Use compact_blank to remove empty strings from Arrays and Hashes"
+description: "This is one of those “Nice Things” in Rails"
+layout: article
+category: ruby
+image:
+  base: '2021/compact_blank-remove-empty-strings-from-array-and-hash'
+  alt: "Compact Disc"
+  credit: "Jan Huber"
+  source: "https://unsplash.com/photos/oTCRizM-PUI"
+
+---
+
+Active Support is a mixed library of “useful” things that’s included with Rails. This includes [extending the base Ruby classes](https://guides.rubyonrails.org/active_support_core_extensions.html) such as `Hash`, `String` and `Integer`.
```

Comment:
*is a Rails library of "useful" things...?

* This includes ways to extend the base Ruby classes, such as...

Reply:
```suggestion
Active Support is a library of “useful” things that supports (!) the other parts of Rails. It includes ways to [extend the base Ruby classes](https://guides.rubyonrails.org/active_support_core_extensions.html), such as `Hash`, `String` and `Integer`.
```